### PR TITLE
Add Java Solution to Usaco guide website for jan 2021 bronze even more odd photos

### PR DESCRIPTION
**Pull request by sathu-beep**
This is a solution to http://www.usaco.org/index.php?page=viewproblem2&cpid=1084 problem 
add all the formatting things like the problem name linked and id cuz idk what that is
I noticed that https://usaco.guide/bronze/intro-greedy?lang=cpp in the usaco guide greedy section this problem only has https://usaco.guide/problem-solutions/ cpp solutions so I wrote one in java to help the java kids understand what was going on. Even in the official usaco solution, there was only one for cpp. This is incredibly useful for java kids as you can see so I would appreciate it if you added it to the website. It works 11/11 test cases you can check. K thx bye

-if you add to github or website pls credit me

### CODE REVIEW

**Comment by dongliu0426** [comment]
please follow step 4 of https://usaco.guide/general/adding-solution?lang=cpp#steps

i think that should fix the failed checks.

**Comment by maggie-j-liu** [comment]
You also need to put the opening `LanguageSection` and `JavaSection` tags.

**Comment by sathu-beep** [comment]
> please follow step 4 of https://usaco.guide/general/adding-solution?lang=cpp#steps
> 
> i think that should fix the failed checks.

k I'm gonna do it rn

File: solutions/USACO jan21-even-more-odd-photos-bronze.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+source: USACO Bronze 2021 January
+title: Even More Odd Photos
+author: Sathvik Chundru
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob2_bronze_jan21.html)
+
+
+</LanguageSection>
+<JavaSection />
```
Comment (dongliu0426) [inline solutions/USACO jan21-even-more-odd-photos-bronze.mdx:12]:
```suggestion
<LanguageSection>

<JavaSection>
```

File: solutions/USACO jan21-even-more-odd-photos-bronze.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      }
+      System.out.println(E + O);
+
+  }
+       
+ 
+}
+```
+
+</JavaSection>
```
Comment (dongliu0426) [inline solutions/USACO jan21-even-more-odd-photos-bronze.mdx:47]:
```suggestion
</JavaSection>

```

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

File: solutions/USACO jan21-even-more-odd-photos-bronze.mdx (hunk 2 above)
Comment (maggie-j-liu) [reply solutions/USACO jan21-even-more-odd-photos-bronze.mdx:47]: technically not necessary but whitespace is good :p

**Comment by maggie-j-liu** [comment]
Follow step 3 of https://usaco.guide/general/adding-solution#steps 
Use tabs to indent and consistently space operators (`a + b` or `a+b` but not both)

File: solutions/USACO jan21-even-more-odd-photos-bronze.mdx (hunk 2 above)
Comment (sathu-beep) [reply solutions/USACO jan21-even-more-odd-photos-bronze.mdx:47]: sus

**Comment by dongliu0426** [comment]
> please follow step 4 of https://usaco.guide/general/adding-solution?lang=cpp#steps
> 
> i think that should fix the failed checks.

@sathu-beep

**Comment by sathu-beep** [comment]
> > please follow step 4 of https://usaco.guide/general/adding-solution?lang=cpp#steps
> > i think that should fix the failed checks.
> 
> @sathu-beep

what but I did  I added that language stuff and the problem id
wbhat else am I missing

**Comment by dongliu0426** [comment]
In the module's .problems.json file (in this case, Binary_Search.problems.json), change the kind of the solutionMetadata field to internal and remove all other fields. Also add tags (if you want). If the problem is not in a module, you can add the problem to extraProblems.json.

```
{
  "uniqueId": "usaco-690",
  "name": "Cow Dance Show",
  "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=690",
  "source": "Silver",
  "difficulty": "Easy",
  "isStarred": false,
  "tags": ["Binary Search", "Ordered Set"],
  "solutionMetadata": {
    "kind": "internal"
  }
},
```
*copied from https://usaco.guide/general/adding-solution?lang=cpp#steps*

**Review by maggie-j-liu — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**